### ISSUE TITLE
Showing Protected SWF file from Amazon S3

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#272]
```
What steps will reproduce the problem?
1. Convert PDF to SWF using swftool.
2. Upload SWF on Amazon S3 and make it protected
3. Take default example of FlexPaper
4. Make a code to get temp URL of SWF which expires in 200 minutes
5. Use it as follows
                var fp = new FlexPaperViewer(   
                         '../FlexPaperViewer',
                         'viewerPlaceHolder', { config : {
                         SwfFile : 'http://bucket.s3.amazonaws.com/folder/1.swf?AWSAccessKeyId=XXXXXXXX&Expires=XXXXXXX&Signature=XXXXXXXx',
                         Scale : 0.6, 
                         ZoomTransition : 'easeOut',
                         ZoomTime : 0.5,
                         ZoomInterval : 0.2,
                         FitPageOnLoad : true,
                         FitWidthOnLoad : false,
                         FullScreenAsMaxWindow : false,
                         ProgressiveLoading : false,
                         MinZoomSize : 0.2,
                         MaxZoomSize : 5,
                         SearchMatchAll : false,
                         InitViewMode : 'SinglePage',

                         ViewModeToolsVisible : true,
                         ZoomToolsVisible : true,
                         NavToolsVisible : true,
                         CursorToolsVisible : true,
                         SearchToolsVisible : true,

                         localeChain: 'en_US'
                         }});

6. I made this in one of the wordpress

What is the expected output? What do you see instead?
Flexpaper should show SWF

What version of the product are you using? On what operating system?
Flexpper 1.5.1, Wordpress, Windows, MySQL, Apache, PHP

Please provide any additional information below.

```

Original issue reported on code.google.com by `saurabhp...@gmail.com` on 21 Apr 2012 at 1:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@devaldi.com` on 24 Apr 2012 at 7:17
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Thanks
Saurabh
```

Original comment by `saurabhp...@gmail.com` on 24 Apr 2012 at 9:11

**Comment by GoogleCodeExporter** [comment]
```
@Saurabh, @Erik

The following might be helpful (Query String Request Authentication):
http://docs.amazonwebservices.com/AmazonS3/latest/dev/RESTAuthentication.html
http://docs.amazonwebservices.com/AmazonS3/latest/dev/S3_QSAuth.html
http://docs.amazonwebservices.com/AWSSDKforPHP/latest/#m=AmazonS3/get_object_url


```

Original comment by `mentur...@gmail.com` on 15 May 2012 at 8:10

**Comment by GoogleCodeExporter** [comment]
```
We are using a (custom) system similar to Amazon AWS. In our case a possible 
solution would be to provide a array of files (one row per page) to SwfFile 
instead of a string.

Another more data-efficient but complex solution would be to provide SwfFile 
with a 'sprintf' format (as in http://nl3.php.net/sprintf ) and provide a 
second array with the values.
```

Original comment by `a.vannat...@gmail.com` on 17 May 2012 at 1:26

**Comment by GoogleCodeExporter** [comment]
```
fixed in upcoming build 1.5.2. Make sure you set EncodeURI to false
```

Original comment by `erik.eng...@devaldi.com` on 12 Jun 2012 at 9:42
- Changed state: **Fixed**
